doc/metric_spec.md

```diff
@@ -102,6 +102,13 @@ The value can vary from 0, 1 , 5 etc to `+Inf`, that  correspond to the infinity
 The number of migration threshold pro node and resource ID set by a pacemaker cluster. 
 Possible values are positive numbers.
 
+### `ha_cluster_pacemaker_config_last_change`
+
+#### Description
+
+This metric has 2 values: the first one is always set to 1 which indicate the presence of the metric. 
+The second one, is the timestamp that correspond to the timestamp updated by pacemaker forresource configuration last change.
+It is useful to see if the resource configuration in the cluster has changed over a period of time and gets alerts.
```

Comment:
I propose rewording this because every metric has the timestamp, and this is no exception. Also, this metric will always be present.

Maybe something like:
```
The relevant part of this metric is its timestamp, which corresponds to the last time Pacemaker configuration changed.
The actual metric value will always be 1 and can be ignored.
```

Reply:
done thx